pkg/kubectl/resource_printer.go

```diff
@@ -270,10 +268,11 @@ func formatEndpoints(endpoints []api.Endpoint) string {
 		return "<none>"
 	}
 	list := []string{}
-	for i := range endpoints {
-		ep := &endpoints[i]
-		list = append(list, net.JoinHostPort(ep.IP, strconv.Itoa(ep.Port)))
-	}
+	//FIXME: What do we want to print, now that endpoints are more complex?
```

Comment:
I think this is used in `kubectl get endpoints` and  `kubectl describe service`.  That's what I've observed working with #4760.

Since both `get endpoints` and `describe service` show terse information about the endpoints, I think a comma-separated list will work, so "protocal://ip:port"?

We should add a new `describe endpoints`  command to show the detailed service multi-port <-> endpoint ports information.  Currently, endpoints do not have describe implemented.